tests/components/selection_input.test.ts

```diff
@@ -263,6 +263,23 @@ describe("Selection Input", () => {
     app.destroy();
   });
 
+  test("Can use arrow keys to change ranges after input corruption", async () => {
+    const { app } = await createSelectionInput({ initialRanges: ["TEST", "C2"] });
+    const selectionInputEl = fixture.querySelectorAll(".o-selection-input input")! as NodeList;
+    focus(0);
+    await nextTick();
+    selectionInputEl[0].dispatchEvent(new KeyboardEvent("keydown", { key: "ArrowLeft" }));
+    await nextTick();
+    expect((selectionInputEl[0] as HTMLInputElement)?.value).toEqual("TEST");
+
+    focus(1);
+    await nextTick();
+    selectionInputEl[1].dispatchEvent(new KeyboardEvent("keydown", { key: "ArrowLeft" }));
+    await nextTick();
+    expect((selectionInputEl[1] as HTMLInputElement)?.value).toEqual("B2");
+    app.destroy();
```

Comment:
Maybe adding app.destroy() was a precautionary step for ensuring proper cleanup post-test execution. I included it because I observed similar cleanup steps in this file at 2-3 other places.

Reply:
If you have a look at the `mountComponent` helper, you will see that we actually register the app to be destroyed at the end of the test :) the only places where it makes sense to destroy the app by yourself are:
- testing a side effect of the destroy/unmount (like in the test `unmounting deletes the state`)
- when you want to mount several components in the same test like in test `The containerRect is the spreadsheet element if it is not explicitly in the popover props` 

Both practices should not be encouraged in general ,it has to stay an exception ^^



* the test `F2 alters edition mode` also does this but its not necessary anymore, probably a forward port that went bad